evaluator/evaluate_test.go

```diff
@@ -79,6 +80,35 @@ func TestEvaluate(t *testing.T) {
 	assert.Equal(t, 2, len(mockWriter.Messages))
 }
 
+func TestEvaluatePruneUpdates(t *testing.T) {
+	assert.NoError(t, os.Setenv("ENABLE_UPDATES_LATEST_ONLY", "true"))
```

Comment:
It will stay set to true after this test. You should set it back, using "defer" e.g.

Reply:
Fixed.